### PR TITLE
[thread-cert] refactor case 5.3.3 using pktverify

### PR DESCRIPTION
**Pull request by FiveDimensions**

### CODE REVIEW

File: tests/scripts/thread-cert/Cert_5_3_03_AddressQuery.py [hunk 1]
```diff
@@ -85,64 +111,36 @@ def test(self):
...
         med1_mleid = self.nodes[MED1].get_ip6_address(config.ADDRESS_TYPE.ML_EID)
         self.assertTrue(self.nodes[ROUTER1].ping(med1_mleid))
 
-        # Verify DUT_ROUTER2 responded with an Address Notification.
-        dut_messages = self.simulator.get_messages_sent_by(DUT_ROUTER2)
-        msg = dut_messages.next_coap_message('0.02', '/a/an')
-        command.check_address_notification(msg, self.nodes[DUT_ROUTER2], self.nodes[ROUTER1])
+        self.collect_ipaddrs()
+        self.collect_rlocs()
+        self.collect_rloc16s()
```
Comment (librasungirl) [inline tests/scripts/thread-cert/Cert_5_3_03_AddressQuery.py:135]: Any specific reasons to put these between pings? How about moving these after topology is formed?

File: tests/scripts/thread-cert/Cert_5_3_03_AddressQuery.py [hunk 2]
```diff
@@ -152,37 +150,211 @@ def test(self):
...
+        #         The DUT MUST receive and process the incoming Address Notification
+        #         The DUT MUST then forward the ICMPv6 Echo Request from MED and
+        #         forward the ICMPv6 Echo Reply to MED
+
+        _pkt = pkts.filter_ping_request().\
+            filter_wpan_src64(MED).\
+            filter_ipv6_dst(ROUTER_3_MLEID).\
+            must_next()
+        pkts.filter_wpan_src64(ROUTER_2).\
+            filter_ipv6_dst(REALM_LOCAL_ALL_ROUTERS_ADDRESS).\
```
Comment (librasungirl) [inline tests/scripts/thread-cert/Cert_5_3_03_AddressQuery.py:213]: reuse `filter_RLARMA()`?

File: tests/scripts/thread-cert/Cert_5_3_03_AddressQuery.py [hunk 3]
```diff
@@ -152,37 +150,211 @@ def test(self):
...
+        _pkt1 = pkts.filter_ping_reply(identifier=_pkt.icmpv6.echo.identifier).\
+            filter_wpan_src64(ROUTER_2).\
+            filter_wpan_dst16(MED_RLOC16).\
+            must_next()
+
+        # Step 5: Power off Router_3 and wait for the Leader to expire its Router ID
+        #         Send an ICMPv6 Echo Request from MED to the Router_3 ML-EID address
+        #         The DUT MUST update its address cache and removes all entries based
+        #         on Router_3’s Router ID.
+        #         The DUT MUST be sent an Address Query to discover Router_3’s RLOC
```
Comment (librasungirl) [inline tests/scripts/thread-cert/Cert_5_3_03_AddressQuery.py:322]:
```suggestion
        #         The DUT MUST send an Address Query to discover Router_3’s RLOC
```

File: tests/scripts/thread-cert/Cert_5_3_03_AddressQuery.py [hunk 4]
```diff
@@ -152,37 +150,211 @@ def test(self):
...
+        #         address.
+
+        _pkt = pkts.filter_ping_request().\
+            filter_wpan_src64(MED).\
+            filter_ipv6_dst(ROUTER_3_MLEID).\
+            filter(lambda p: p.sniff_timestamp - _pkt1.sniff_timestamp >= 700).\
+            must_next()
+        pkts.filter_wpan_src64(ROUTER_2).\
+            filter_ipv6_dst(REALM_LOCAL_ALL_ROUTERS_ADDRESS).\
```
Comment (librasungirl) [inline tests/scripts/thread-cert/Cert_5_3_03_AddressQuery.py:331]: reuse `filter_RLARMA()`

File: tests/scripts/thread-cert/Cert_5_3_03_AddressQuery.py [hunk 5]
```diff
@@ -152,37 +150,211 @@ def test(self):
...
+        #         Send two ICMPv6 Echo Requests from Router_1 to MED ML-EID
+        #         The DUT MUST NOT respond with an Address Notification message
+
+        pkts.filter_ping_request().\
+            filter_wpan_src64(ROUTER_1).\
+            filter_ipv6_dst(MED_MLEID).\
+            must_next()
+
+        pkts.filter_wpan_src64(ROUTER_1).\
+            filter_ipv6_dst(REALM_LOCAL_ALL_ROUTERS_ADDRESS).\
```
Comment (librasungirl) [inline tests/scripts/thread-cert/Cert_5_3_03_AddressQuery.py:348]: reuse `filter_RLARMA()`

File: tests/scripts/thread-cert/Cert_5_3_03_AddressQuery.py (hunk 1 above)
Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_5_3_03_AddressQuery.py]: moved to the topology formed

**Review by librasungirl — APPROVED** [review]

**Review by jwhui — APPROVED** [review]

**Pull request merged**